Dockerfile

```diff
@@ -15,6 +15,11 @@ SHELL ["/bin/bash", "-o", "pipefail", "-c"]
 RUN curl -sSL https://raw.githubusercontent.com/python-poetry/poetry/master/get-poetry.py | python - \
     && poetry config virtualenvs.create false
 
+# Install libvips
+RUN rm -rf /var/lib/apt/lists/* \
+    apt-get update \
+	&& apt-get install -y  --no-install-recommends "libvips-dev=8.10.5-2"
```

Comment:
Should not this be in the [base docker image](https://github.com/Nautilus-Cyberneering/librarian-system-dockerfile/blob/main/Dockerfile)?

Reply:
Actually, it is already there. Removed from here.